src/schemas/index.ts

```diff
@@ -0,0 +1,10 @@
+import * as types from '@nordicsemiconductor/lwm2m-types'
+
+// For these Objects we use the definition provided by @nordicsemiconductor/lwm2m-types
+export const ConnectivityMonitoring_4_urn = types.ConnectivityMonitoring_4_urn
+export const Device_3_urn = types.Device_3_urn
+export const Humidity_3304_urn = types.Humidity_3304_urn
+export const Location_6_urn = types.Location_6_urn
+export const parseURN = types.parseURN
```

Comment:
`parseURN` function was removed from this file because the intention of this file is to clarify the version of objects used and that function has no relation with that topic. It can be imported from ` @nordicsemiconductor/lwm2m-types` as normal.

Reply:
Comments were added to make it even clearer what version is being used. 

Type definitions are now also exported because the URN and the type definition are correlated with the version used.  

https://github.com/MLopezJ/asset-tracker-lwm2m-js/blob/3bc425bc297bb23d0bafc9f9576d562c9751bc9e/src/schemas/index.ts